site/cache.md

```diff
@@ -40,13 +41,23 @@ Commands:
 
 - `addPatch`: Add a patch to the cache. This command verifies if the path points to a valid patch by querying the Oracle support portal.
     ```
-    imagetool cache addPatch --type wls --version 12.2.1.3.0 --user abc@xyz.com --passwordEnv MYVAR --patchId p12345678 --path /path/to/patch.zip
+    imagetool cache addPatch --type wls --version 12.2.1.3.0  --patchId 12345678 --path /path/to/patch.zip
     ```
+Note:  When adding a patch to the cache store. The patchId should be in the following format:  99999999_9.9.9.9.99999  The first 8 digits is the patch id, followed by an underscore and then release number.  This is needed if you want to distinguish a patch that has different versions of the patch.  
 
-- `getCacheDir` and `setCacheDir`: Get or set the cache directory. Used to display or set the directory where patches will be downloaded.
+For example, patch 29135930 has different versions from Oracle support.
```

Comment:
For example, patch 29135930 has several different versions from Oracle support, one for each release where the bug is fixed.